src/components/Quote.vue

```diff
@@ -144,50 +144,17 @@ export default {
 
 			return subtitle
 		},
-
-		shortenQuotedMessage() {
-			let cutAtChar = 200
-
-			// We allow a maximum of 2 line breaks
-			const firstNewline = this.simpleQuotedMessage.indexOf('\n')
-			if (firstNewline !== -1) {
-				const secondNewline = this.simpleQuotedMessage.indexOf('\n', firstNewline + 1)
-				if (secondNewline !== -1) {
-					cutAtChar = Math.min(cutAtChar, secondNewline)
-				}
-			}
-
-			let cuttingAfterWord = true
-			if (cutAtChar <= this.simpleQuotedMessage.length) {
-				// When we cut the string, we look for the next space.
-				const nextSpace = this.simpleQuotedMessage.indexOf(' ', cutAtChar)
-				if (nextSpace !== -1 && nextSpace < cutAtChar + 15) {
-					// If it is in a reasonable distance, we finish the word and cut after it
-					cutAtChar = nextSpace
-				} else {
-					// If not, we look if there is a space reasonable before the cut position
-					const previousSpace = this.simpleQuotedMessage.lastIndexOf(' ', cutAtChar)
-					if (previousSpace !== -1 && previousSpace > cutAtChar - 15) {
-						cutAtChar = previousSpace
-					} else {
-						// We didn't find any space in near distance, so we cut
-						// just where we are and add the … at the position,
-						// since we cut in the middle of the word.
-						cuttingAfterWord = false
-					}
-				}
+		// Shorten the message to 250 characters and append three dots to the end of the
+		// string. This is needed because on very wide screens, if the 250 characters
+		// fit, the css rules won't ellipsize the text-overflow.
+		shortenedQuoteMessage() {
+			if (this.simpleQuotedMessage.length >= 250) {
+				return this.simpleQuotedMessage.substring(0, 250) + '...'
```

Comment:
Please use the ellipsis character instead (for Jan)

Reply:
OK! I didn't know there was a difference, thanks :)